dbt/__init__.py

```diff
@@ -1,6 +0,0 @@
-# N.B.
```

Comment:
pretty sure we need these files...

Reply:
Yeah. Agreed. I removed these for debugging purposes while working on this a couple months ago, but will make sure these are not deleted in the final Pr.